MaterialDesignThemes.Wpf/Themes/Generic.xaml

```diff
@@ -770,7 +755,7 @@
                                         x:Name="ContentPresenter" Opacity="1"                    
                                         Content="{TemplateBinding Content}" ContentTemplate="{TemplateBinding ContentTemplate}" ContentStringFormat="{TemplateBinding ContentStringFormat}" />
                             </AdornerDecorator>
-                            <Grid x:Name="PART_ContentCover" Background="{x:Null}" Opacity="0" IsHitTestVisible="False" Focusable="False" />
+                            <Grid x:Name="PART_ContentCover" Background="{TemplateBinding OverlayBackground}" Opacity="0" IsHitTestVisible="False" Focusable="False" />
```

Comment:
This is good, but there is one thing missing. When the `PART_ContentCover` is not visible, we need to set the Visibility to Collapsed or the Visual Studio Live Visual Tree wont be able to select elements under it.

This is similar to the trigger that is done in the DialogHost [here](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/master/MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.DialogHost.xaml#L179-L187).
I believe all that we need to do is copy that grid style in here too so that the grid is collapsed when it is not visible.